README.md

```diff
@@ -51,27 +51,16 @@ v1.0 - Intial release for iOS with full support of all operations based on plan
 
 ```shell
 npm install --save react-native-sqlite-storage
+rnpm link
 ```
 
-#### Step 2. XCode SQLite project dependency set up
-
-Drag the SQLite Xcode project as a dependency project into your React Native XCode project
-
-![alt tag](https://raw.github.com/andpor/react-native-sqlite-storage/master/instructions/libs.png)
-
-#### Step 3. XCode SQLite libraries dependency set up
-
-Add libSQLite.a (from Workspace location) to the required Libraries and Frameworks. Also add sqlite3.0.tbd (XCode 7) or libsqlite3.0.dylib (XCode 6 and earlier) in the same fashion using Required Libraries view (Do not just add them manually as the build paths will not be properly set)
```

Comment:
Are you sure that your script is actually handling the project linkage and  libSQLite.a steps? I think I recognized the Frameworks add for sqlite library only?